pkg/virtualKubelet/forge/forge_test.go

```diff
@@ -39,3 +39,25 @@ var _ = Describe("Virtual Kubelet labels test", func() {
 	})
 
 })
+
+var _ = Describe("Forge toleration test", func() {
+	Context("Tolerations", func() {
+		tol1 := corev1.Toleration{
+			Key:      "virtual-node.liqo.io/not-allowed",
+			Operator: "Exist",
+			Effect:   "NoExecute",
+		}
+		tol2 := corev1.Toleration{
+			Key:      "kubernetes.azure.com/scalesetpriority",
+			Operator: "Equal",
+			Value:    "spot",
+			Effect:   "NoSchedule",
+		}
```

Comment:
Variables should be populated in BeforeEach blocks, to prevent side-effects. Additionally, would avoid mentioning specific cloud providers if not necessary

Reply:
Thanks!